### ISSUE TITLE
Extension Is Disappearing Every Day Or So!

### ISSUE THREAD

**Issue by sapote2241** [iamadamdev/bypass-paywalls-chrome#1075]
I recently bought a new Chromebox 3. The current build on the new device is Chrome OS Version 87.0.4280.88 (Official Build) (64-bit).
 
The old device can't be updated beyond Chrome OS Version 76.0.3809.136
(Official Build) (64-bit). I had been using bypass-paywalls-chrome without a problem on the old device, using version 1.7.6.

After updating the new device to 87.0.4280.88, and sideloading version 1.7.8 on the device, I'm noticing the extension disappears from my Extensions page every day or two! Has Google, in build 87.0.4280.88, blocked extensions from being sideloaded?  Or, at least, begun deleting them every day or so?

I just noticed on my extensions page that uBlock Origin wasn't enabled. But every site that I visited with bypass paywalls did have uBlock Origin enabled.

**Comment by iamadamdev** [comment]
Not sure, don't have a chromebox to test.

**Comment by sapote2241** [comment]
I'm shocked!  Two hours ago, I deleted all my files in Google Drive
and re-installed this extension.  I used it for 15 minutes. No
problem. I just checked my list of extensions, and bypass paywalls
(1.7.8) has disappeared again! I have my account on this Chromebox
specifically NOT sync'ing with any other device that I have. Right
now, I have to re-pack the extension every time I want to use it!
It's simple enough, but I shouldn't have to!  I don't know what the
problem is. I'm going to try power washing the new Chromebox that I
bought (delete my account from it).
